src/components/pages/annual-report-2018/AnnualReportView.js

```diff
@@ -0,0 +1,331 @@
+            housing and benefits, schedule appointments with doctors and case
+            managers, search for jobs, and stay in touch with family and
+            friends. It&apos;s a lifeline for getting back on your feet.
+          </p>
+          <p className="annual-report--text">
+            ShelterConnect works with internet service providers to offer free
+            wifi in shelters and transitional housing facilties and offers
+            portable chargers to address battery charging issues.
+          </p>
+          <div className="annual-report--shelterconnect-stats">
+            <h2>In 2018, ShelterConnect expanded wifi installations by 228%</h2>
+            <ul>
+              <li>
+                <span className="goal-number">3000</span>
+                <p>
+                  people now have access to wifi via ShelterConnect, annually
+                </p>
+              </li>
+              <li>
+                <span className="goal-number">600</span>
+                <p>people no have access to wifi via ShelterConnect, nightly</p>
+              </li>
+              <li>
+                <span className="goal-number">200</span>
+                <p>portable phone chargers distributed</p>
+              </li>
+              <li>
+                <span className="goal-number">7</span>
+                <p>
+                  new wifi installations, including 3 new emergency shelters
+                </p>
+              </li>
+              <li>
+                <span className="goal-number">$50</span>
+                <p>cost for wifi in a shelter for an entire month</p>
+              </li>
+              <li>
+                <span className="goal-number">$0.02</span>
+                <p>cost for wifi for someone in a shelter per night</p>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div className="annual-report--askdarcel">
+        <div className="wrapper">
+          <img src={ImgAskDarcelLogo} alt="AskDarcel" />
+          <p className="annual-report--text">
+            AskDarcel is a robust and continously updated online directory for
+            case managers, city employees, social workers, and people in the
+            homeless, low/no income, at risk, and supportive communities to find
+            housing and human services in San Francisco.
+          </p>
+          <p className="annual-report--text">
+            In 2018, we released an Alpha of the product but decided to hold off
+            on a public launch while we validated the content on the site—trust
+            is the biggest concern for people experiencing homelessness when it
+            comes to referrals for services. The good news is we’ve now
+            validated everything on the site and are ready for our Beta launch.
+          </p>
+          <h3>
+            In 2018, over <span className="white">2,000</span> hours were spent
+            on the development of AskDarcel.
```

Comment:
Here's also a spot where both this PR and master show "AskDarcel" but the actual live site shows "SF Service Guide".